corporate/views/billing_page.py

```diff
@@ -139,7 +142,7 @@ def billing_home(request: HttpRequest) -> HttpResponse:
                 publishable_key=STRIPE_PUBLISHABLE_KEY,
                 stripe_email=stripe_customer.email,
                 CustomerPlan=CustomerPlan,
-                onboarding=request.GET.get("onboarding") is not None,
+                onboarding=onboarding,
```

Comment:
Where is the code that sets generates URLs with `?onboarding=`?  We're changing this to expect `true` specifically, and I'm not sure how to check if that's correct.

Reply:
```
$ git grep onboarding=
corporate/tests/test_stripe.py:            self.assertEqual(response.url, "/billing/?onboarding=true")
corporate/views/billing_page.py:                onboarding=onboarding,
corporate/views/upgrade.py:            billing_page_url = f"{billing_page_url}?onboarding=true"
zerver/tests/test_signup.py:        self.assertEqual(result.url, "http://zuliptest.testserver/upgrade/?onboarding=true")
zerver/views/auth.py:            redirect_to = "{}?onboarding=true".format(reverse("initial_upgrade"))
```
We are always passing `onboarding=true` in the URL. I did a lookup in desktop and mobile on Github, and there is no relevant matches.

Reply:
Yeah. That is correct. Though it looks like the `onboarding=true` param goes missing when we redirect to /billing page from upgrade page. This is a bug in master. I think a side effect of the stripe migration prep commits that started accepting a  succes callback function for `helpers.create_ajax_request` . Anyway not that important since this applies only for free trial and the flow is changed completely in the new version.